includes/controllers/class.llms.controller.certificates.php

```diff
@@ -117,10 +117,33 @@ public function maybe_handle_reporting_actions() {
 			$this->download( $cert_id );
 		} elseif ( isset( $_POST['llms_delete_cert'] ) ) {
 			$this->delete( $cert_id );
+		} elseif ( isset( $_POST['llms_enable_cert_sharing'] ) || isset( $_POST['llms_disable_cert_sharing'] ) ) {
+			$this->change_sharing_settings( $cert_id, isset( $_POST['llms_enable_cert_sharing'] ) );
 		}
 
 	}
 
+	/**
+	 * Change shareable settings of a certificate
+	 *
+	 * @since [version]
+	 *
+	 * @param int  $cert_id WP Post ID of the llms_my_certificate
+	 * @param bool $is_allowed Allow share the certificate or not
+	 * @return void
+	 */
+	private function change_sharing_settings( $cert_id, $is_allowed ) {
+
+		$cert = new LLMS_User_Certificate( $cert_id );
+
+		if ( ! $cert->can_user_manage() ) {
+			return;
+		}
+
+		update_post_meta( $cert_id, '_llms_allow_sharing', $is_allowed ? 'yes' : 'no' );
```

Comment:
Can use certificate setter in favor of directly using postmeta apis (this will make this automatically take advantage of built-in LifterLMS hooks).

```suggestion
		$cert->set( 'allow_sharing', $is_allowed ? 'yes' : 'no' );
```